pkg/apis/networking/v1/utils.go

```diff
@@ -62,6 +64,9 @@ func ValidateAddressRange(ar *AddressRange) (err error) {
 	if len(ar.End) > 0 && !cidr.Contains(end) {
 		return fmt.Errorf("end %s is not in CIDR %s", ar.End, ar.CIDR)
 	}
+	if len(ar.Start) > 0 && len(ar.End) > 0 && ip.Cmp(start, end) > 0 {
+		return fmt.Errorf("start %s > end %s", start, end)
```

Comment:
could you use some words instead of symbol `>` ?

Reply:
Greater? Like 
![image](https://user-images.githubusercontent.com/17826874/133241221-2a2a1363-d1bf-486a-a1c1-da7e986a5865.png)

Reply:
eg.
```
return fmt.Errorf("subnet should have at least one available IP")
```

Reply:
got u